packages/go/analysis/ad/esc6.go

```diff
@@ -34,7 +50,43 @@ func PostCanAbuseUPNCertMapping(_ context.Context, _ graph.Database, operation a
 									ToID:   dcForNode.ID,
 									Kind:   ad.CanAbuseUPNCertMapping,
 								}) {
-									return fmt.Errorf("context timed out while creating CanAbuseUPNCert edge")
+									return fmt.Errorf("context timed out while creating CanAbuseUPNCertMapping edge")
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+		return nil
+	})
+	return nil
+}
+
+func PostCanAbuseWeakCertBinding(_ context.Context, _ graph.Database, operation analysis.StatTrackedOperation[analysis.CreatePostRelationshipJob], enterpriseCertAuthorities []*graph.Node) error {
+	operation.Operation.SubmitReader(func(ctx context.Context, tx graph.Transaction, outC chan<- analysis.CreatePostRelationshipJob) error {
+		for _, eca := range enterpriseCertAuthorities {
+			if ecaDomainSID, err := eca.Properties.Get(ad.DomainSID.String()).String(); err != nil {
```

Comment:
I'm not sure if we should bomb out of the entire analysis if we fail to process one node here